src/shared/components/markdown-view/markdown-view.tsx

```diff
@@ -0,0 +1,251 @@
+import * as React from 'react';
+import cx from 'classnames';
+import * as _ from 'lodash-es';
+import { Converter } from 'showdown';
+import * as sanitizeHtml from 'sanitize-html';
+import { useTranslation } from 'react-i18next';
+import { useForceRender, useResizeObserver } from '../../hooks';
+
+import './_markdown-view.scss';
```

Comment:
Rename files to PascalCase.
```suggestion
import './MarkdownView.scss';
```